dcrsqlite/apisource.go

```diff
@@ -706,14 +706,14 @@ func makeExplorerAddressTx(data *dcrjson.SearchRawTransactionsResult, address st
 	tx.Confirmations = data.Confirmations
 
 	for i := range data.Vin {
-		if data.Vin[i].PrevOut.Addresses != nil {
+		if data.Vin[i].PrevOut.Addresses != nil && len(data.Vin[i].PrevOut.Addresses) != 0 {
```

Comment:
What I meant by "It is OK to check the length of a nil size" is that you can omit the first check and only verify length >0

Reply:
https://play.golang.org/p/dhx3OR9C5v

Reply:
Alright, I'll get on that.